Update nested-virtualization.md

andrewallen opened the pull request:
Changed New-VMSwitch -SwitchName to New-VMSwitch -Name as per correct syntax. https://docs.microsoft.com/en-us/powershell/module/hyper-v/new-vmswitch?view=win10-ps

azurecla commented:
Hi __@BrucePerlerMS__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (bruceper). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

BryanLa commented:
Hey @BrucePerlerMS - I can't merge this one due to merge conflicts :-/  Likely caused by the other PR on this same file that was just merged. LMK if you'd like to close and resubmit, or open a PR against the private repo.

BrucePerlerMS commented:
Hmmm. I’ll have a look and likely cancel and resubmit.
Thanks,
Bruce

From: Bryan Lamos [mailto:notifications@github.com]
Sent: Tuesday, September 22, 2015 9:16 AM
To: Azure/azure-content <azure-content@noreply.github.com>
Cc: Bruce Perler <Bruce.Perler@microsoft.com>
Subject: Re: [azure-content] Update key-vault-developers-guide.md (#4565)


Hey @BrucePerlerMS<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fBrucePerlerMS&data=01%7c01%7cBruce.Perler%40microsoft.com%7cddbb13a8df9e4673a09008d2c36921a6%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=Bm7TWaRcX8IybQvfYuBNeODwZlVJPmWY9RfDtWw6CAU%3d> - I can't merge this one due to merge conflicts :-/ Likely caused by the other PR on this same file that was just merged. LMK if you'd like to close and resubmit, or open a PR against the private repo.

—
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fAzure%2fazure-content%2fpull%2f4565%23issuecomment-142337367&data=01%7c01%7cBruce.Perler%40microsoft.com%7cddbb13a8df9e4673a09008d2c36921a6%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=UZ3%2barL1TngUx9ZbUIvP6euE6fx8T9Q1aGR5QKUuQqw%3d>.

BrucePerlerMS commented:
I’d like to close and resubmit. ☺

From: Bryan Lamos [mailto:notifications@github.com]
Sent: Tuesday, September 22, 2015 9:16 AM
To: Azure/azure-content <azure-content@noreply.github.com>
Cc: Bruce Perler <Bruce.Perler@microsoft.com>
Subject: Re: [azure-content] Update key-vault-developers-guide.md (#4565)


Hey @BrucePerlerMS<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fBrucePerlerMS&data=01%7c01%7cBruce.Perler%40microsoft.com%7cddbb13a8df9e4673a09008d2c36921a6%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=Bm7TWaRcX8IybQvfYuBNeODwZlVJPmWY9RfDtWw6CAU%3d> - I can't merge this one due to merge conflicts :-/ Likely caused by the other PR on this same file that was just merged. LMK if you'd like to close and resubmit, or open a PR against the private repo.

—
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fAzure%2fazure-content%2fpull%2f4565%23issuecomment-142337367&data=01%7c01%7cBruce.Perler%40microsoft.com%7cddbb13a8df9e4673a09008d2c36921a6%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=UZ3%2barL1TngUx9ZbUIvP6euE6fx8T9Q1aGR5QKUuQqw%3d>.

BrucePerlerMS commented:
merge conflict suggests I try again

andrewallen committed:
Update nested-virtualization.md

Changed New-VMSwitch -SwitchName to New-VMSwitch -Name as per correct syntax. https://docs.microsoft.com/en-us/powershell/module/hyper-v/new-vmswitch?view=win10-ps
